### PR TITLE
Kses - Part 7

### PR DESCRIPTION
**Pull request by hoseinrafiei**
This is related to https://github.com/eventespresso/event-espresso-core/issues/3738 issue.

Applied `wp_kses` function to all variables in **/modules** directory.

Here is the testing checklist: https://trello.com/c/bml8JS9m/1156-ee4-core-testing-checklist-pr-3803

### CODE REVIEW

File: modules/bot_trap/EED_Bot_Trap.module.php [hunk 1]
```diff
@@ -120,14 +121,13 @@ public function run($WP)
      */
     public static function generate_bot_trap()
     {
-        $do_not_enter = esc_html__('please do not enter anything in this input', 'event_espresso');
-        $time = esc_attr(microtime(true));
+        $time = microtime(true);
         $html = '<div class="tkt-slctr-request-processor-dv" style="float:left; margin:0 0 0 -999em; height: 0;">';
-        $html .= '<label for="tkt-slctr-request-processor-email-' . $time . '">' . $do_not_enter . '</label>';
+        $html .= '<label for="tkt-slctr-request-processor-email-' . esc_attr($time) . '">' . esc_html__('please do not enter anything in this input', 'event_espresso') . '</label>';
```
Comment (tn3rb) [inline modules/bot_trap/EED_Bot_Trap.module.php:126]: 👍🏻

**Review by tn3rb — APPROVED** [review]
Great Job 🏅 🥇

**Comment by hoseinrafiei** [comment]
@Saam01 and @mohdshakeel

Please test this PR carefully in both of caff and decaf versions. Note that you should check `fix/kses-7` branch.
You should check everything but following sections should be more focused.

- The invoice
- The receipt
- iCal
- Ticket selector
- Entire registration process
-- Attendee info
-- Payments
-- Thank you page

**Comment by mohdshakeel** [comment]
We are working on it and finished almost 50% work.

**Comment by mohdshakeel** [comment]
This issue still coming https://github.com/eventespresso/event-espresso-core/issues/3792.

**Comment by Saam01** [comment]
Got issue with backend registration: #3804

**Comment by hoseinrafiei** [comment]
> This issue still coming #3792.

No problem. It fixed and merged into master.

**Comment by Saam01** [comment]
Issue with Attendee mover: #3807

**Comment by hoseinrafiei** [comment]
> Issue with Attendee mover: #3807

Thanks for reporting this but it's a general issue and is not related to this branch.
I just confirmed with master branch.

**Comment by hoseinrafiei** [comment]
@garthkoyle 

Do you think that I can merge this PR? The checklist is fully checked and no related issue were detected.

- #3792 is already fixed and merged into master.
- #3804 which seems to be a wrong report and is closed now.
- #3807 which is a general issue replicable even in master.

**Comment by garthkoyle** [comment]
I don't have any information to prohibit the merge. Mo and Sam said they finished the testing. 

So, @mohdshakeel and @Saam01 in the future, if you don't find any issues after testing, please report that to the ticket so the developer knows that they can proceed.

**Comment by mohdshakeel** [comment]
Yes, We will report here.

**Comment by Saam01** [comment]
Sure @garthkoyle.

**Pull request merged**